ogticrd/covid-safe-paths
# Feat: notify covid positive user

#### Description:
Adding a pop up to notify a positive covid user about laboratories results, saved the user covidId in a list in the cellphone storage along with useType of the form ("mySelf", "others"), fetching the user state from firebase, also deleted the way of save the user info in the storage 



#### Linked issues:


Fix: [17](https://trello.com/c/DQsEzTAi/17-as-a-develover-i-need-a-notification-fail-proff-covid-id-positive-afected-service-3)

#### Screenshots:


![image](https://user-images.githubusercontent.com/43874147/89051123-c45c8280-d321-11ea-9c02-8e91b4a0c75a.png)

#### How to test:


- `git pull && git checkout Feat/17-notify-covid-positive-user`
- You got to change to TRUE the ***checkState*** variable in the code to get yourself as positive covid user  https://github.com/intellisysdcorp/covid-safe-paths/blob/7e6f89886f1d0cd5a98280e2eb8d2c1eca66a864/app/views/DR/HomeScreen/index.js#L140-L142
- fill the form with a negative id
- close and open the app
- In the dashboard screen it should display a dialog info notify you that you are positive covid
- Then it will move you to positive onboarding screen to enter a nickname and ask you to share your location
